deltacat/compute/compactor_v2/steps/merge.py

```diff
@@ -161,6 +161,7 @@ def _merge_tables(
 
     final_table = pa.concat_tables(result_table_list)
     final_table = final_table.drop([sc._PK_HASH_STRING_COLUMN_NAME])
+    final_table = final_table.sort_by([("dw_last_updated", "descending")])
```

Comment:
Can we only do this if dw_last_updated is present in the schema?

Reply:
Sounds good, that will make the change potentially safe to merge.

Reply:
Yes, can we also add a TODO here to retrieve sort order policy from the table version metadata instead of hard-coding here?

Reply:
We probably need a test case with dw_last_updated column and ensure it passes too.

Reply:
Addressed comments. Added unit test of sort order, which passes.